test/helpers/DeckBuilder.js

```diff
@@ -88,13 +88,13 @@ class DeckBuilder {
 
     getNamedCardsInPlayerEntry(playerEntry) {
         let namedCards = [];
-        if (typeof playerEntry === 'number' || typeof playerEntry == null) {
+        if (typeof playerEntry === 'number' || typeof playerEntry === 'boolean' || playerEntry === null) {
```

Comment:
Now that I'm looking at it, these should probably just throw TestSetupError

Reply:
I almost started to do that but the function calling getNamedCardsInPlayerEntry is looping through the entire Player Options object and catching any setting, this was to filter out things w/o breaking. 

I can work on getting that calling function to use only specific fields rather than the entire object, then we can make this throw an error.

Reply:
Ah right, so number specifically is because some fields can pass it to "pad" the particular area. Changed it around slightly but should handle these better now.

Reply:
Ahhh forgot there was a legit case for that, my bad. Much easier to read now either way, thanks for handling that!